src/common/metrics.rs

```diff
@@ -194,6 +209,20 @@ impl MetricsProvider for CollectionsTelemetry {
             MetricType::GAUGE,
             vec![gauge(total_optimizations_running as f64, &[])],
         ));
+
+        metrics.push(metric_family(
+            "collection_points",
```

Comment:
We might need to name it like this?

```suggestion
            "collection_points_total",
```

We can decide on that separately. I'd be fine with adjusting all metric names at the end in a single PR to follow convention. Then we can review together with some of our cloud people whom are more familiar with this.

Reply:
According to the prometheus docs https://prometheus.io/docs/instrumenting/writing_exporters/
> _total is a convention for counters, you should use it if you’re using the COUNTER type.

`collection_points` is a gauge, not counter type so per naming convention, this should not have a `_total` suffix.
If we already use this suffix for gauges in other places, I'm fine with adding this suffix to make it consistent.